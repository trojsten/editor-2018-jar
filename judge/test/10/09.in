INT:
velkostmapy
20
INT:
dlzka
0
INT:
vyska
0
INT:
okuliare
0
VECTOR_STR:
mapa
20
.HN/W0>+V%A+##L0D=*/
5Y3=RVL+'3SQ##MO2F18
9=9NRH,'1UM2##8FT64U
KGAUYFLYWWM[##M/Z13,
;D.?[T:N?(%0##)I(%CK
P)&U-MJ,9NAI##</STZ9
S%R/(R1CJ&Y%##W$ES*1
D$14-N/4A>Z+##9O5E=3
3CD3KSP>;NM3##V2&*UJ
OH*B5U:1WM8Y##=SW&K[
,Q[P)X9?9B[9##KR%B5S
LD(J;4SSH6UJ##;>R03F
<;HD:Y,QE$%J##BW>+9P
AU.F=0L(%2CY##T&[+U-
+:2(C0FH-$W<##G,+'60
T?M5*[M+3XLI##=>J@&4
-4*+&54V,K$2##0PI,3%
=SY8CJU8QW/+##-=BZ*M
B9PW=6HO=>5L##T=1+UX
1)%-6AS+6SWCI=$:&08>
