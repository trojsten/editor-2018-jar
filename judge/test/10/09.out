{"dlzka": 2, "vyska": 19}